services/core-web/src/components/mine/ProjectSummaries/ProjectSummary.js

```diff
@@ -94,8 +94,9 @@ export class ProjectSummary extends Component {
   handleUpdate = (message) => {
     const mineGuid = this.props.match?.params?.mineGuid;
     const projectSummaryGuid = this.props.match?.params?.projectSummaryGuid;
+    const projectGuid = this.props.formValues.project_guid;
```

Comment:
Because we don't have Project setup in minespace yet I've had to do some work arounds until that work is done. This is meant only as a temporary measure.